test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -209,6 +209,7 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 
 	supportedGpuTypes := []string{"nvidia-tesla-k80", "nvidia-tesla-v100", "nvidia-tesla-p100"}
 	for _, gpuType := range supportedGpuTypes {
+		gpuType := gpuType // create new variable for each iteration step
```

Comment:
This does not make any sense.

Reply:
Actually it does. I am not golang fan :)

https://stackoverflow.com/questions/26692844/captured-closure-for-loop-variable-in-go